paddle/math/tests/test_CpuGpuVector.cpp

```diff
@@ -84,4 +84,10 @@ int main(int argc, char** argv) {
   return ret;
 }
 
+#else
```

Comment:
This lines may be can removed in mac osx.
Since the `whole-archive` was not correct before, this lines ware added.

I will give a PR to you, and check it is cool or not.